package.json

```diff
@@ -2,10 +2,10 @@
   "name": "creditcard.js",
   "version": "2.1.3",
   "description": "A simple library for credit-card validation in JavaScript",
-  "main": "src/creditcard.js",
+  "main": "dist/creditcard.min.js",
```

Comment:
@fernahh It is not better to make the package point to the unminified version of the library and leave the build system of the end-user (like webpack) take care from the minification process?

Reply:
yeah, it makes sense :thinking: